runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/ExecutableStageDoFnOperator.java

```diff
@@ -52,19 +60,22 @@
 import org.apache.beam.sdk.values.PCollectionView;
 import org.apache.beam.sdk.values.TupleTag;
 import org.apache.beam.sdk.values.WindowingStrategy;
+import org.apache.flink.api.java.functions.KeySelector;
 import org.apache.flink.streaming.runtime.streamrecord.StreamRecord;
 import org.joda.time.Instant;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
- * ExecutableStageDoFnOperator basic functional implementation without side inputs and user state.
- * SDK harness interaction code adopted from {@link
- * org.apache.beam.runners.flink.translation.functions.FlinkExecutableStageFunction}. TODO: Evaluate
- * reuse All operators in the non-portable streaming translation are based on {@link DoFnOperator}.
- * This implies dependency on {@link DoFnRunner}, which is not required for portable pipeline. TODO:
- * Multiple element bundle execution The operator (like old non-portable runner) executes every
- * element as separate bundle, which will be even more expensive with SDK harness container.
+ * ExecutableStageDoFnOperator basic functional implementation. SDK harness interaction code adopted
```

Comment:
We can scratch this (since it is no longer relevant) and should also update rest of the JavaDoc block (I initially thought you did, but this looks like formatting only).

Reply:
Will refine this comment.